index.js

```diff
@@ -1 +1,51 @@
-// Rachelle Pestanas & Brett Sullivan
\ No newline at end of file
+// Rachelle Pestanas & Brett Sullivan
+const redBlock = document.getElementsByClassName("block--red")[0];
+const blueBlock = document.getElementsByClassName("block--blue")[0];
+const greenBlock = document.getElementsByClassName("block--green")[0];
+const pinkBlock = document.getElementsByClassName("block--pink")[0];
+const grayBlock = document.getElementsByClassName("block--gray")[0];
+const blockParent = document.getElementsByClassName("blocks")[0];
+redBlock.addEventListener("click", (event) => {
```

Comment:
Using ```querySelectorAll()``` could have helped you not repeat yourself so much throughout the rest of the code. Since all the blocks are being referenced in one variable ```rockets```, you could have then iterated through each block and performed a certain action: ```rockets.forEach(rocket => {
  rocket.addEventListener('click', () => {
    const parent = rocket.parentNode;
    console.log('parent', parent);
    parent.removeChild(rocket);
    parent.prepend(rocket);
  });
});```